bills-app/src/main/webapp/index.html

```diff
@@ -26,13 +26,19 @@ <h1>Contxt</h1>
             <p>Informing today's generation about the country's laws</p>
          </section>
 
+         <section class = "search-bar">
+             <p>Search for bills!</p>
+            <input id="searchbutton" placeholder="Search for..."></input>
+            <button value="input" onclick="serchBillsWithAPI()">Search</button>
```

Comment:
super nit, just a minor type: serchBillsWithApi() --> searchBillsWithApi() here and elsewhere